pipelines/nextflow/modules/schema/check_json_schema.nf

```diff
@@ -24,8 +24,8 @@ process CHECK_JSON_SCHEMA {
         tuple val(meta), path(json_file), emit: verified_json
 
     shell:
-        schema = params.json_schemas[json_file.baseName]
+        schema_name = json_file.baseName
```

Comment:
```suggestion
        schema_name = json_file.simpleName
```
mb use `getSimpleName` instead of `getBaseName `  to exclude file extension as well? 
https://www.nextflow.io/docs/latest/script.html#getting-file-attributes

Reply:
Since there is only one extension, `baseName` works for this case, but I feel safer with `simpleName`, cheers!